src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,39 @@
+.calendar {
+  width: $calendar-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 20px;
+
+  @each $day, $margin in $days {
+    &--start-day--#{$day} &__day:first-child {
+      margin-left: $margin;
+    }
+  }
+
+  @for $i from 28 to 31 {
+    &--month-lenght--#{$i} :nth-child(n+#{$i+1}) {
+      display: none;
+    }
+  }
+  @for $day from 1 through 31 {
+    &__day:nth-child(#{$day})::before {
+      box-sizing: border-box;
+      content: "#{$day}";
```

Comment:
<img width="797" alt="image" src="https://user-images.githubusercontent.com/102411130/233926399-3b839e04-85ee-4128-886d-020beae5c705.png">

Reply:
leave only `content: "#{$day}";` in loop

no need to repeat all styles